src/layouts/layout.jsx

```diff
@@ -59,11 +60,43 @@ export const StyledFooterText = styled.label`
 
 function Layout({ children }) {  
   const [contextState, setContextState] = useState(initialState);
+  const [ handleMenu, setHandleMenu ] = useState(false);
+  const isMobile = useMediaQuery("(max-width: 768px)");
+
+  const handleMenuNavShows = () => {
+    if (isMobile === false) {
+      return <NavMenu />
+    }
+    return;
+  }
```

Comment:
I think you can avoid calling the `handleMenuNavShows` when the `ismobile` is true

Reply:
I don't understand.

Reply:
See here: https://github.com/xmartlabs/xl-blog/pull/31#discussion_r1101673496